design.md

```diff
@@ -0,0 +1,30 @@
+# Design
+
+Visually communicate a message or action with text, images, and shapes.
+
+## Beginning Design
+
+* Take thoughtbot's [Design for Developers tutorial](http://upcase.com/design-for-developers)
```

Comment:
They are called video tutorials now. But I think this should be “Watch” instead of “Take”

> On Jan 30, 2015, at 1:59 PM, Derek Prior <notifications@github.com> wrote:
> 
> In design.md <https://github.com/thoughtbot/trail-map/pull/166#discussion_r23863309>:
> 
> > @@ -0,0 +1,30 @@
> > +# Design
> > +
> > +Visually communicate a message or action with text, images, and shapes.
> > +
> > +## Beginning Design
> > +
> > +* Take thoughtbot's [Design for Developers tutorial](http://upcase.com/design-for-developers)
> workshop?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/thoughtbot/trail-map/pull/166/files#r23863309>.
> 

---
Chad Pytel, Founder and CEO
t: 617-482-1300 x113
f: 866-217-5992
http://thoughtbot.com 
http://twitter.com/thoughtbot